numpy/core/defchararray.py

```diff
@@ -1797,6 +1817,8 @@ def isdecimal(a):
     For each element, return True if there are only decimal
     characters in the element.
 
+    Decimal characters are those that can be used to form numbers in base 10.
```

Comment:
The examples look reasonably solid to me--I did wonder if this extra sentence added much vs. the one we already have below with:

> Decimal characters include digit characters, and all characters that can be used to form decimal-radix numbers, e.g. U+0660, ARABIC-INDIC DIGIT ZERO.

Reply:
Gotcha, I've removed it! I mistakenly thought decimal numbers were those with a decimal point so I added that line but I must have missed the existing explanation.